rosapi/interface.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+
+class Interface():
+  def __init__(self, api, interface):
+    self.interface_name = interface
+    self.api = api
+
+  def checkinterface(self):
```

Comment:
it does if we're 
1. trying to be consistent
2. not reinventing the wheel from upstream.  

If you go to the cli to display anything at a path, the command is always "print".